config.yaml

```
@@ -32,5 +32,12 @@ auths:
   smpPassword: "password"
   pwrCrtlAuth: "user:password"
   camAuth: "user:password"
+matrix:
+  username: "rbgbot"
+  password: "password"
+  homeserver: "matrix.org"
+  roomId: "roomid"
+
```

Just a thought on future extensibility

 
Something like this:
```suggestion
alerts:
  matrix:
   username: "rbgbot"
   password: "password"
   homeserver: "matrix.org"
   roomId: "roomid"
```

With a corresponding config struct like this one:
```go
Alerts struct {
		Matrix *struct {
			Username   string `yaml:"username"`
			Password   string `yaml:"password"`
			Homeserver string `yaml:"homeserver"`
			RoomID     string `yaml:"roomId"`
		} `yaml:"matrix"`
		Email *struct {
			Address string `yaml:"address"`
			Name  string `yaml:"name"`
		} `yaml:"email"`
	} `yaml:"alerts"`
```
would allow optional other strategies for future use. (Note that email is a pointer which defaults to nil if not configured)

---

Alerts could then be a pointer as well? Correct me if I'm wrong though :D

---

Nice idea, structuring the config like this is really a lot cleaner. :+1:
Yes, a pointer seems to work in this case.